kDrive/UI/View/Header view/FilesHeaderView.swift

```diff
@@ -75,11 +82,43 @@ class FilesHeaderView: UICollectionReusableView {
         let tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(didTapOnCard))
         uploadCardView.addGestureRecognizer(tapGestureRecognizer)
 
+        setupTrashView()
+
         selectView = SelectView.instantiate()
         selectView.isHidden = true
         containerStackView.addArrangedSubview(selectView)
     }
 
+    private func setupTrashView() {
+        trashInformationView.isHidden = true
+
+        trashInformationTitle.font = .systemFont(ofSize: 14)
+        trashInformationTitle.textColor = KDriveResourcesAsset.headerTitleColor.color
+        trashInformationSubtitle.font = .systemFont(ofSize: 14)
+        trashInformationSubtitle.textColor = KDriveResourcesAsset.infomaniakColor.color
```

Comment:
You should use IKLabel with TextStyle. TextStyle provides it's own color maybe you will have to still apply a custom color afterwards.